apps/meteor/server/settings/file-upload.ts

```diff
@@ -25,6 +25,11 @@ export const createFileUploadSettings = () =>
 			i18nDescription: 'FileUpload_MediaTypeBlackListDescription',
 		});
 
+		await this.add('FileUpload_UnknownMediaTypeProtection', true, {
```

Comment:
I think it would be better to rename this to something like `Protect_...`

Reply:
We decided not to have a setting anymore. Having a fallback media type will be the default behavior